desktop/release-notes.md

```diff
@@ -24,12 +24,95 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.15.0
+
+{% include release-date.html date="2022-12-01" %}
```

Comment:
@maxcleme the release-date.html page needs to be updated with the latest download packages and checksums!

Also, the individual buttons on the Linux install packages will need changing as well

Reply:
@aevesdocker Yes, I'm waiting for the final RC right now, it should updated as soon as I get everything lined up. I'll keep this thread opened until it's fixed. Thanks for the reminder!

Reply:
Should be good!